server/src/server.ts

```diff
@@ -0,0 +1,45 @@
+import { Socket, Server } from "socket.io";
+
+export class CallForHelpServer {
+    public static readonly PORT: number = 3000;
+    private port: string | number;
+    private io: Server;
+
+    constructor() {
+        this.configurate();
+        this.createServerSocket();
+        this.listen();
+    }
+
+    private configurate(): void { 
+        const proccessPort = process.env.PORT;
+
+        if (proccessPort === undefined) {
+            this.port = CallForHelpServer.PORT;
+            console.log(`Environment variable port is undefined, defaulting to ${CallForHelpServer.PORT}`);
+        } else {
+            this.port = proccessPort;
+        }
+    }
+
+    private createServerSocket(): void {
+        this.io = require('socket.io')(this.port, {
```

Comment:
If you're going to use ES modules go all the way, don't use both

https://socket.io/docs/v3/server-initialization/#typescript